contracts/deploys.go

```diff
@@ -0,0 +1,78 @@
+package contracts
+
+import (
+	"testing"
```

Comment:
The package is for production apps, while we get `testing`, `neotest` and other things here.

Reply:
Moved